src/lib/leagues/masterTasks.ts

```diff
@@ -0,0 +1,1065 @@
+		}
+	},
+	{
+		id: 4011,
+		name: 'Receive every Revenant weapon',
+		has: async ({ cl }) => {
+			return resolveItems(["Craw's bow (u)", "Thammaron's sceptre (u)", "Viggora's chainmace (u)"]).every(gs =>
+				cl.has(gs)
+			);
+		}
+	},
+	{
+		id: 4012,
+		name: 'Achieve base level 110 stats',
+		has: async ({ skillsLevels }) => {
+			return Object.values(skillsLevels).every(i => i >= 110);
+		}
+	},
+	{
+		id: 4013,
+		name: 'Create every draconic shield',
+		has: async ({ cl }) => {
+			return resolveItems(['Dragonfire ward', 'Dragonfire shield', 'Ancient wyvern shield']).every(gs =>
+				cl.has(gs)
+			);
+		}
+	},
+	{
+		id: 4014,
+		name: 'Create every nightmare staff',
+		has: async ({ cl }) => {
+			return resolveItems([
+				'Volatile nightmare staff',
+				'Harmonised nightmare staff',
+				'Eldritch nightmare staff'
+			]).every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 4015,
+		name: 'Achieve a TzKal cape',
+		has: async ({ cl }) => {
+			return cl.has('TzKal cape');
+		}
+	},
+	{
+		id: 4016,
+		name: 'Create every piece of Zenyte jewellery',
+		has: async ({ cl }) => {
+			return resolveItems(['Zenyte ring', 'Zenyte necklace', 'Zenyte bracelet', 'Zenyte amulet']).every(gs =>
+				cl.has(gs)
+			);
+		}
+	},
+	{
+		id: 4017,
+		name: 'Create every piece of enchanted Zenyte jewellery',
+		has: async ({ cl }) => {
+			return resolveItems([
+				'Ring of suffering',
+				'Necklace of anguish',
+				'Tormented bracelet',
+				'Amulet of torture'
+			]).every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 4018,
+		name: 'Finish the god wars CL',
+		has: async ({ cl }) => {
+			return godWarsDungeonCL.every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 4021,
+		name: 'Complete the Inferno 30 Times',
+		has: async ({ activityCounts }) => {
+			return (activityCounts.Inferno ?? 0) >= 30;
+		}
+	},
+	{
+		id: 4022,
+		name: 'Smelt 1000 Runite bars from scratch',
+		has: async ({ smeltingStats }) => {
+			return smeltingStats.amount('Runite bar') >= 2000;
+		}
+	},
+	{
+		id: 4023,
+		name: 'Build a Ancient rejuvenation pool in your PoH',
+		has: async ({ poh }) => {
+			return poh.pool === 99_950;
+		}
+	},
+	{
+		id: 4024,
+		name: 'Complete 500 slayer tasks',
+		has: async ({ slayerTasksCompleted }) => {
+			return slayerTasksCompleted >= 500;
+		}
+	},
+	{
+		id: 4025,
+		name: 'Do 2000 laps at the Ardougne Rooftop Course',
+		has: async ({ lapScores }) => {
+			return (lapScores[11] ?? 0) >= 2000;
+		}
+	},
+	{
+		id: 4026,
+		name: 'Do 1000 laps at the Prifddinas Rooftop Course',
+		has: async ({ lapScores }) => {
+			return (lapScores[13] ?? 0) >= 1000;
+		}
+	},
+	{
+		id: 4027,
+		name: 'Do 1000 laps at the Daemonheim Rooftop Course',
+		has: async ({ lapScores }) => {
+			return (lapScores[30] ?? 0) >= 1000;
+		}
+	},
+	{
+		id: 4028,
+		name: 'Obtain 5000 Marks of grace',
+		has: async ({ cl }) => {
+			return cl.amount('Mark of grace') >= 5000;
+		}
+	},
+	{
+		id: 4030,
+		name: 'Reach total level 3000',
```

Comment:
Says total level 3000 but only checks for 2800. Change one or the other.

_(Maybe there should be both 2800 and 3000 tasks. The next highest i believe is 2400/2500)_